Unable to build Core Systems (sheltekio/icmp4j  issue)

On a fresh system I'm unable to build the `core-java-spring` repo from the `master branch`. 

*Steps to reproduce:* 

To reproduce on your already existing build system: 
* remove the `sheltekio` folder under `~/.m2/repositories/`

On a new system:
1. clone the repo
2. running `mvn clean install -DskipTests` results in

```
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] core-java-spring                                                   [pom]
[INFO] Core Common                                                        [jar]
[INFO] Arrowhead Service Registry                                         [jar]
[INFO] Arrowhead Authorization                                            [jar]
[INFO] Arrowhead Certificate Authority                                    [jar]
[INFO] Arrowhead Orchestrator                                             [jar]
[INFO] Arrowhead Event Handler                                            [jar]
[INFO] Relay Library                                                      [jar]
[INFO] Arrowhead Gatekeeper                                               [jar]
[INFO] Arrowhead Gateway                                                  [jar]
[INFO] Arrowhead Choreographer                                            [jar]
[INFO] Arrowhead System Registry                                          [jar]
[INFO] Arrowhead Device Registry                                          [jar]
[INFO] Arrowhead Onboarding Controller                                    [jar]
[INFO] Arrowhead QoS Monitor                                              [jar]
[INFO] arrowhead-installer                                                [pom]
[INFO] docker-all                                                         [pom]
[INFO] 
[INFO] -------------------------< eu.arrowhead:core >--------------------------
[INFO] Building core-java-spring 4.2.0                                   [1/17]
[INFO] --------------------------------[ pom ]---------------------------------
Downloading from central: https://repo1.maven.org/maven2/sheltekio/javalibs/icmp4j/1.2/icmp4j-1.2.pom
Downloading from github: https://maven.pkg.github.com/arrowhead-f/client-library-java-spring/sheltekio/javalibs/icmp4j/1.2/icmp4j-1.2.pom
Downloading from spring-repo: https://repo.spring.io/plugins-release/sheltekio/javalibs/icmp4j/1.2/icmp4j-1.2.pom
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for core-java-spring 4.2.0:
[INFO] 
[INFO] core-java-spring ................................... FAILURE [  2.507 s]
[INFO] Core Common ........................................ SKIPPED
[INFO] Arrowhead Service Registry ......................... SKIPPED
[INFO] Arrowhead Authorization ............................ SKIPPED
[INFO] Arrowhead Certificate Authority .................... SKIPPED
[INFO] Arrowhead Orchestrator ............................. SKIPPED
[INFO] Arrowhead Event Handler ............................ SKIPPED
[INFO] Relay Library ...................................... SKIPPED
[INFO] Arrowhead Gatekeeper ............................... SKIPPED
[INFO] Arrowhead Gateway .................................. SKIPPED
[INFO] Arrowhead Choreographer ............................ SKIPPED
[INFO] Arrowhead System Registry .......................... SKIPPED
[INFO] Arrowhead Device Registry .......................... SKIPPED
[INFO] Arrowhead Onboarding Controller .................... SKIPPED
[INFO] Arrowhead QoS Monitor .............................. SKIPPED
[INFO] arrowhead-installer ................................ SKIPPED
[INFO] docker-all ......................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.946 s
[INFO] Finished at: 2020-11-20T11:32:58+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project core: Could not resolve dependencies for project eu.arrowhead:core:pom:4.2.0: Failed to collect dependencies at sheltekio.javalibs:icmp4j:jar:1.2: Failed to read artifact descriptor for sheltekio.javalibs:icmp4j:jar:1.2: Could not transfer artifact sheltekio.javalibs:icmp4j:pom:1.2 from/to spring-repo (https://repo.spring.io/plugins-release/): Authentication failed for https://repo.spring.io/plugins-release/sheltekio/javalibs/icmp4j/1.2/icmp4j-1.2.pom 401 Unauthorized -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

I looked at [mvncentral](https://mvnrepository.com/artifact/sheltekio.javalibs/icmp4j/1.2) and attempting to download the jar from there, ends up in a login/password request.
The linked [Github repo](https://www.github.com/sheltekio/icmp4j) does not exist anymore. 

Following the same [artifact link](https://repo.spring.io/plugins-release/sheltekio/javalibs/icmp4j/1.2/icmp4j-1.2.pom) results in the same login/password request.

--- a/pom.xml
+++ b/pom.xml
@@ -72,7 +72,12 @@
             <name>Spring Repository</name>
 			<url>https://repo.spring.io/plugins-release/</url>
         </repository>
-    </repositories>
+		<repository>
+			<id>jenkins-repo</id>
+			<name>Jenkins Repository</name>
+			<url>https://repo.jenkins-ci.org/public/</url>
+		</repository>
+	</repositories>
 
 	<dependencies>
 		<dependency>

